alembic/operations/base.py

```diff
@@ -1247,6 +1251,10 @@ def create_table(
              quoting of the schema outside of the default behavior, use
              the SQLAlchemy construct
              :class:`~sqlalchemy.sql.elements.quoted_name`.
+            :param if_not_exists: If True, adds IF NOT EXISTS operator when
+             creating the table.
+
+             .. versionadded:: 1.13.4
```

Comment:
Not really sure if this is the correct next version. Happy to change it.

Reply:
it would be 1.13.3 since it's yet to be released